c7n/filters/core.py

```diff
@@ -459,7 +458,10 @@ def process_value_type(self, sentinel, value):
                 sentinel = datetime.now(tz=tzutc()) + timedelta(sentinel)
 
             if not isinstance(value, datetime):
-                value = parse(value, default=datetime.now(tz=tzutc()))
+                try:
```

Comment:
I normalized this one to look the same as the one above, and also filter for the 3 error types.